src/cascade/executor/epiviz_runner.py

```diff
@@ -169,6 +170,37 @@ def add_omega_constraint(model_context, execution_context, sex_id):
     model_context.input_data.observations = pd.concat([observations, asdr], ignore_index=True, sort=True)
 
 
+def add_age_steps(model_context):
+    smallest_step = model_context.parameters.ode_step_size
+    added = compute_age_steps(smallest_step)
+    model_context.parameters.additional_ode_steps = added
+
+
+def compute_age_steps(smallest_step):
+    """We will add age steps to whatever they have given. The GBD chooses
```

Comment:
"they"? I assume you mean "the modelers"? I think it's better to talk about where the thing we're adding to actually comes from, the settings or the model_context depending on how you want to think about it.